std_msgs/Header header

uint32 received_msg_size # number of received message size in bytes 
uint32 transmitted_msg_size # number of transmitted message size in bytes